.travis.yml

```diff
@@ -2,38 +2,81 @@ language: python
 
 sudo: false
 
+env:
+  global:
+    - secure: "ZNlSgPnERbar1rPNd3tjUtsPry+BrPCjA0l1AP7n3Iz11EOrTRAv6tjv5lV8Q3LqeRA9iLCpGiku1+niCNI1vYGk0P1UlBh5mzMbrSeL5tqz1pdRSS9QUPoT05QaY4S+/WmPoG61Pju+i29rYsFsxzGMbrJtypysTtrJLFIGE/A="
```

Comment:
I registered `doctr`, a tool to auto-upload the docs to `gh-pages`, on my fork/account. If you can grant me adim rights to the repo I can update this to the main repo.

Reply:
> If you can grant me adim rights to the repo I can update this to the main repo.

I don't think I can give any more rights than a collaborator. 
![Apr15::075840](https://user-images.githubusercontent.com/3688009/56110321-4c175580-5f54-11e9-92e6-df0f2e7a0a5d.png)

Reply:
Let me re-crete that token and test.

Reply:
Nope. I cannot write an SSH key:

```
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.github.com/repos/wesleybowman/UTide/keys
```

However, we don't have too! [`doctr` has an option to set it up without admin rights](https://drdoctr.github.io/recipes.html#setting-up-doctr-for-a-repo-you-don-t-have-admin-access-to). You will need to do this part though:

```
================== You should now do the following ==================

1. Go to https://github.com/wesleybowman/UTide/settings/keys
   and add the following as a new key:

    ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC5mHCsE7RBjRjkBWSTI1hKXEok83R4YdLqk963K6zhQYN04g1QbvZOMNNSgvgrOgXOWZRvlop/URQMyH5oGB9Ul1954mZMc7p13rkszfXWPaNe6NvUcTagZ5Xik0WWbVPSMOBrWsCohqDcflZzmccu4RJA2Nx1iAskD6cDi8D5NvVQC1+W8JAdwEdJ7jCJayuTYyJRdfPQPYGEoIUlml7cOP9Dw3KIK8263vQaT36yhA+Dilum1VdmD7GCevKHMO8emzTdDBMMlFWjntMDkilsg5wpsXok53n5dw3uwnQg71avMjzIdoX10fZRp5m2Q68hLRIzAj4caYdR72xlR+/JjpM5KeOSxAKyMXgFnvBfO0umeaIQ0uy5auzBvKoW+rL/YNPmi4XzuQyYSgoAGMY9decK8Mvnt4imz/56PH3K6Sjbx+DDNfC1moH0oRjuARxE4jMJLOwEYbls40CkLLQQuA8xJaiHHkOI4IG0uL8FoInbxmBz3JG3TaRxoGPqQeYRGYy06N3WnqJRZ7wZoRNEhUZuKyw3uZSoCM5zmatyMrcjeiFvsGkNedAMZUiMGf3a0LQJxkO1RW3Mgwyt5EwK8bezNzPeyazpcXpzwbUdGxlHQeKNOKh1idS3Z+k/vMYI9RmvVCJRuQGl9fp4wec2jVAv0iOpmSxx3tbUYQxXMQ==
   Be sure to allow write access for the key.
```

Reply:
Done!

Reply:
Should be OK to merge now.